include/z64camera.h

```diff
@@ -439,7 +439,7 @@ typedef struct {
 typedef struct {
     /* 0x00 */ SwingAnimation swing;
     /* 0x1C */ f32 unk_1C;
-    /* 0x20 */ VecSph unk_20;
+    /* 0x20 */ VecGeo unk_20;
```

Comment:
unk_20 is unused isn't it?
is there a hint it's even a struct and not separate members?

Reply:
```c
rwData->unk_20.pitch = 0;
rwData->unk_20.yaw = 0xC8;
```
and
`rwData->unk_20.r = eyeAtOffset.r;`
Seems to be the only place it's set, and remains unused. I guess it was just previously inferred from the `.r` that it combined into a `VecSph` during decomp. `0xC8` seems to be an odd yaw though, so I could see it being its own variable